doc/man7/RAND.pod

```diff
@@ -54,6 +54,13 @@ only in exceptional cases and is not recommended, unless you have a profound
 knowledge of cryptographic principles and understand the implications of your
 changes.
 
+Finally, it is possible for a provider to bypass the default RAND setup for
+L<RAND_bytes(3)> and associated functions.  A provider can specified as the
+single randomness source via the L<RAND_set1_random_provider(3)> or via
```

Comment:
Either: "via RAND_set1_random_provider(3)" or "via the RAND_set1_random_provider(3) function"